velox/docs/functions/spark/string.rst

```diff
@@ -44,6 +44,18 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT lower('SparkSql'); -- sparksql
 
+.. spark:function:: lpad(string, len, pad) -> string
+    
+    Returns ``string``, left-padded with pad to a length of ``len``. If ``string`` is
+    longer than ``len``, the return value is shortened to ``len`` characters or bytes.
+    If ``pad`` is not specified, ``string`` will be padded to the left with space characters
+    if it is a character string, and with zeros if it is a byte sequence. ::
```

Comment:
> if it is a character string, and with zeros if it is a byte sequence.

What does "if it is a byte sequence" means? Is there a test that illustrates this case?

Reply:
I think ```SELECT hex(lpad(unhex('aabb'), 5)); ----000000AABB``` is the case.  It doesn't seem to be added to the test, I will add it.

Reply:
After comparing the introduction of the two engines, I found that not only the parameters are different, but further implementation is needed, and I will finish the modification in two days.